crates/bevy_gltf/src/loader.rs

```diff
@@ -1251,12 +1354,30 @@ fn load_node(
                     let primitive_label = primitive_label(&mesh, &primitive);
                     let bounds = primitive.bounding_box();
 
-                    let mut mesh_entity = parent.spawn(PbrBundle {
-                        // TODO: handle missing label handle errors here?
-                        mesh: load_context.get_label_handle(&primitive_label),
-                        material: load_context.get_label_handle(&material_label),
-                        ..Default::default()
-                    });
+                    let mut mesh_entity = if primitive
+                        .extensions()
+                        .map(|extensions| extensions.contains_key("BEVY_meshlet_mesh"))
+                        .unwrap_or(false)
+                    {
+                        if cfg!(feature = "meshlet") {
+                            parent.spawn(MaterialMeshletMeshBundle::<StandardMaterial> {
+                                // TODO: handle missing label handle errors here?
+                                meshlet_mesh: load_context.get_label_handle(&primitive_label),
+                                material: load_context.get_label_handle(&material_label),
+                                ..Default::default()
+                            })
+                        } else {
+                            panic!("glTF file contained a MeshletMesh, but the meshlet cargo feature is not enabled.")
+                        }
+                    } else {
+                        parent.spawn(PbrBundle {
+                            // TODO: handle missing label handle errors here?
```

Comment:
Don't think so. There shouldn't ever be a missing label error, as they're created earlier in the function. This is also an existing comment, it just looks new because I moved the code blocks around a bit.

Reply:
Right you are, missed that this was part of the move.